components/org.wso2.carbon.identity.oauth.endpoint/src/main/java/org/wso2/carbon/identity/oauth/endpoint/state/OAuthRequestStateValidator.java

```diff
@@ -114,15 +114,6 @@ private void validateRequest(OAuthMessage oAuthMessage)
                 oAuthMessage.setSessionDataKeyFromConsent(null);
             }
         }
-
-        validateOauthApplication(oAuthMessage);
-    }
-
-    private void validateOauthApplication(OAuthMessage oAuthMessage) throws InvalidRequestParentException {
```

Comment:
Any reason why we removed this method?

Reply:
With the modifications(as stated in the description) we no longer need this validation method as the clientId validation will be done at the time of app validation.